test/celeritas/ext/ImportData.test.cc

```diff
@@ -0,0 +1,768 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/ImportData.test.cc
+//---------------------------------------------------------------------------//
+#include "celeritas/io/ImportData.hh"
+
+#include <algorithm>
+#include <unordered_map>
+
+#include "corecel/Types.hh"
+#include "corecel/cont/Range.hh"
+#include "celeritas/ext/GeantImporter.hh"
+#include "celeritas/ext/GeantSetup.hh"
+#include "celeritas/io/ImportPhysicsTable.hh"
+#include "celeritas/mat/MaterialView.hh"
+#include "celeritas/phys/CutoffView.hh"
+#include "celeritas/phys/PDGNumber.hh"
+
+#include "celeritas_test.hh"
+
+using namespace celeritas;
+
+//---------------------------------------------------------------------------//
+// TEST HARNESS
+//---------------------------------------------------------------------------//
+/*!
+ * \note
+ * G4EMLOW7.12 and G4EMLOW7.13 produce slightly different physics vector
+ * values for steel, failing \c processes test.
+ */
+class ImportDataTest : public celeritas_test::Test
```

Comment:
Since this is done as part of the geant importer, my first thought is that it should be part of that test

Reply:
We can. I have no strong feelings about this, so I'm happy to move them back, but my reasoning for having 2 tests is:
- GeantImporter: test the different loading options (DataSelection; GeantPhysicsOptions)
- ImportData: test the fully-loaded data (so we will always run it with all options on)
- A bit easier to read/manage the tests, since the ImportData alone is already pretty long.

Reply:
The third point is true, but I'm generally opposed to breaking up tests into multiple files because they're getting long.  I think you can also reduce the test size by dropping the tests of the particle/process/model names since those are being tested.

The GeantImport class isn't just to test the different loading options, it's there to test the full functionality of the geant import class (and all of the things it calls). Indeed you end up having to duplicate its test harness for the new class.

The `ImportData` objects have no functionality to speak of so don't need their own tests.